test/common/cdp.py

```diff
@@ -209,6 +262,10 @@ def start(self):
                 src = f.read()
             # HACK: injecting sizzle fails on missing `document` in assert()
             src = src.replace('function assert( fn ) {', 'function assert( fn ) { return true;')
+            # HACK: sizzle tracks document and when we switch frames, it sees the old document
+            # although we execute it in different context.
```

Comment:
Meh, about time we finally kick out sizzle.. Most of our selectors should be just fine with standard CSS selectors now, and we can add some helpers to test-functions.js for the one or two remaining use cases that still need sizzle. `</thinking_aloud>`

Reply:
yeah, I tried to kick out sizzle, see https://github.com/cockpit-project/cockpit/pull/13015